### PR TITLE
Document sanity development.

### PR DESCRIPTION
**Pull request by kravvcuCL**
Provide guide on how to run sanity tests without the packaging and
docker image building jobs but by using existing images.

### CODE REVIEW

File: source/user/sanity-development.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+them time-consuming. However, it is possible to use existing images of contrail components and run
+only the sanity job by adding the `contrail_docker_registry` and `contrail_version` vars to the
+job vars definition. The changes suggested below should only be done for testing changes in the sanity
+job steps and should be reverted before performing the merge. Let's assume changes are needed for
+`contrail-sanity-centos7-kolla-queens` job. Introduce the following changes:
+
+1. Decide on the docker registry to be used (most likely ci-repo.englab.juniper.net with IP of: 10.84.5.81).
+2. Find the contrail version to use:
+
+  * Execute: `curl http://10.84.5.81:5010/v2/contrail-vrouter-kernel-init/tags/list`
```
Comment (jluk-codi) [inline source/user/sanity-development.rst]: Remove this step which cannot be executed by external users.

File: source/user/sanity-development.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  This will provide all the tags available for the given repository (here: contrail-vrouter-kernel-init).
+  It is best to use the newest available tag e.g. 5.0-278 or master-313
+
+3. Adjust the definition of the job using the chosen values (contrail-zuul-jobs project). The full definition would be:
```
Comment (jluk-codi) [inline source/user/sanity-development.rst]: Before the full definition, add only the fragment with `contrail_docker_registry` and `contrail_version vars`, so the example can be easily used in other jobs.

File: source/user/sanity-development.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+job steps and should be reverted before performing the merge. Let's assume changes are needed for
+`contrail-sanity-centos7-kolla-queens` job. Introduce the following changes:
+
+1. Decide on the docker registry to be used (most likely ci-repo.englab.juniper.net with IP of: 10.84.5.81).
+2. Find the contrail version to use:
+
+  * Execute: `curl http://10.84.5.81:5010/v2/contrail-vrouter-kernel-init/tags/list`
+
+  This will provide all the tags available for the given repository (here: contrail-vrouter-kernel-init).
+  It is best to use the newest available tag e.g. 5.0-278 or master-313
```
Comment (jluk-codi) [inline source/user/sanity-development.rst]: Maybe suggest using `latest`?

**Review by jluk-codi — CHANGES_REQUESTED** [review]

File: source/user/sanity-development.rst (hunk 1 above)
Comment (kravvcuCL) [reply source/user/sanity-development.rst]: Done.

File: source/user/sanity-development.rst (hunk 3 above)
Comment (kravvcuCL) [reply source/user/sanity-development.rst]: Done.

File: source/user/sanity-development.rst (hunk 2 above)
Comment (kravvcuCL) [reply source/user/sanity-development.rst]: Done.

**Pull request closed without merging**